### DISCUSSION TITLE
iSH on iOS 10.3.3

### DISCUSSION THREAD

**Discussion by madmongoose** [ish-app/ish#2196] [Q&A] [answered]
Hello! First of all - thanks a lot for iSH. It's one of the best apps on iOS.

So, I successfully use iSH in my fresh iPhone and iPad, but the story is about another device. I found my old iPhone 5c, and trying to breathe a second life into it. Of course, the first thing I tried jailbreak and installed a lot of apps. Problem with iSH that can be installed only on iOS 11 and higher. But iPhone 5c has 10.3.3. I tried to change minimum iOS version in info.plist, but it didn't help. 

Is it worth trying to build the iSH from src or is it doomed to failure and iSH will not work on 10.3.3?

**Comment by saagarjha** [comment, marked as answer]
iSH does not support 32-bit platforms, unfortunately.

**Reply by madmongoose** [reply]
Thanks for answer! You saved my time)

**Reply by ghost** [reply]
> iSH does not support 32-bit platforms, unfortunately.
